src/UI/Component/Popover/Popover.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace ILIAS\UI\Component\Popover;
+
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Signal;
+use ILIAS\UI\Component\Triggerable;
+use ILIAS\UI\Implementation\Component\Popover\ReplaceContentSignal;
+
+/**
+ * Describes the Popover component
+ */
+interface Popover extends Component, Triggerable {
+
+	/**
+	 * Get the title of the popover.
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Get the components representing the content of the popover.
+	 *
+	 * @return Component[]
+	 */
+	public function getContent();
+
+	/**
+	 * Get the position of the popover.
+	 *
+	 * @return string
```

Comment:
By default the position is `auto` so there are actually 3 positions.

> The renderer acts on a concrete implementation, correct?
Not really, it accepts any implementation of a `Popover`. At least this seems the convention of the framework. This fact also indicates that the constants should go to the interface. I will move them.